examples/kitty/board/qemu_virt_aarch64/kitty.system

```diff
@@ -0,0 +1,324 @@
+    <memory_region name="eth_tx_buffer_data_region_micropython" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="eth_rx_buffer_data_region_micropython" size="0x200_000" page_size="0x200_000" />
+
+    <!-- shared memory for eth driver/eth virt queue mechanism -->
+    <memory_region name="eth_rx_free_drv" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_active_drv" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_tx_free_drv" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_tx_active_drv" size="0x200_000" page_size="0x200_000"/>
+
+    <!-- shared memory for eth virt_rx/eth copy queue mechanism -->
+    <memory_region name="eth_rx_free_copy_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_active_copy_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_free_copy_micropython" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_active_copy_micropython" size="0x200_000" page_size="0x200_000"/>
+
+    <!-- shared memory for eth copy/client queue mechanism -->
+    <memory_region name="eth_rx_free_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_active_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_free_micropython" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_rx_active_micropython" size="0x200_000" page_size="0x200_000"/>
+
+    <!-- shared memory for client/virt_tx queue mechanism -->
+    <memory_region name="eth_tx_free_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_tx_active_nfs" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_tx_free_micropython" size="0x200_000" page_size="0x200_000"/>
+    <memory_region name="eth_tx_active_micropython" size="0x200_000" page_size="0x200_000"/>
+
+    <!-- serial data regions -->
+    <memory_region name="serial_tx_data_driver" size="0x4_000" />
+    <memory_region name="serial_tx_data_micropython" size="0x2_000" />
+    <memory_region name="serial_tx_data_nfs" size="0x2_000" />
+    <memory_region name="serial_rx_data_driver" size="0x2_000" />
+    <memory_region name="serial_rx_data_micropython" size="0x2_000" />
+
+    <!-- shared memory for serial driver/serial virt queue mechanism -->
+    <memory_region name="serial_rx_queue_driver" size="0x1_000" />
+    <memory_region name="serial_tx_queue_driver" size="0x1_000" />
+
+    <!-- shared memory for serial virt/client queue mechanism -->
+    <memory_region name="serial_tx_queue_micropython" size="0x1_000" />
+    <memory_region name="serial_rx_queue_micropython" size="0x1_000" />
+    <memory_region name="serial_tx_queue_nfs" size="0x1_000" />
+
+    <!-- shared memory for micropython/nfs queue -->
+    <memory_region name="shared_nfs_micropython" size="0x200_000" />
```

Comment:
The shared NFS region is a lot larger on the Odroid-C4 Kitty. Is there any reason why this doesn't follow that?

Reply:
Same with `nfs_command_queue` and `nfs_completion_queue`.

Reply:
No reason, this was updated recently with the odroidc4, ill mirror the changes for qemu.